tensorly/contrib/mps_decomposition_cross.py

```diff
@@ -0,0 +1,428 @@
+import tensorly as tl
+from ..mps_tensor import mps_to_tensor
+from ..random import check_random_state
+import numpy as np
+
+rng = check_random_state(1)
```

Comment:
I had missed that. The rng shouldn't be set globally.
Better to add a parameter `random_seed=None` in the core funciton.
Then, inside that function you can use 
`rng = check_random_state(random_seed)`.

Reply:
Oops..
I moved it to random_seed=None; rng = check_random_state(random_seed) inside the function.
Otherwise, we can have random_seed as an input, as in candecomp_parafac.